py_trees/display.py

```diff
@@ -270,17 +293,27 @@ def add_edges(root, root_dot_name, visibility_level, collapse_decorators):
             return
         if visibility_level < root.blackbox_level:
             for c in root.children:
-                (node_shape, node_colour, node_font_colour) = get_node_attributes(c, visibility_level)
-                proposed_dot_name = c.name
-                while proposed_dot_name in names:
-                    proposed_dot_name = proposed_dot_name + "*"
-                names.append(proposed_dot_name)
-                node = pydot.Node(proposed_dot_name, shape=node_shape, style="filled", fillcolor=node_colour, fontsize=fontsize, fontcolor=node_font_colour)
+                (node_shape, node_colour, node_font_colour) = get_node_attributes(c)
+                node_name = c.name
+                while node_name in names:
+                    node_name += "*"
+                names.append(c.name)
+                node = pydot.Node(
+                    node_name,
+                    label=get_node_label(node_name, c),
+                    shape=node_shape,
+                    style="filled",
+                    fillcolor=node_colour,
+                    fontsize=fontsize,
+                    fontcolor=node_font_colour,
+                    # tooltip="foo",
```

Comment:
please delete commented code.

Reply:
I'll switch it to a different comment. Would like to keep them around so I don't forget those attributes are available. The pydot documentation is a bit of a rabbit hole.

Reply:
Done